code/modules/mob/living/living_defines.dm

```diff
@@ -159,7 +159,7 @@
 	//this stuff is here to make it simple for admins to mess with custom held sprites
 	var/icon/held_lh = 'icons/mob/pets_held_lh.dmi'//icons for holding mobs
 	var/icon/held_rh = 'icons/mob/pets_held_rh.dmi'
-	var/icon/head_icon = 'icons/mob/pets_held.dmi'//what it looks like on your head
+	var/icon/head_icon = 'icons/mob/animal_item_head.dmi'//what it looks like on your head
```

Comment:
Because you changed this line to have `animal_item_head.dmi` be the default value for `head_icon`, you can remove this whole line from `frog.dm`
https://github.com/tgstation/tgstation/blob/80d4f95239b7643975f82a0576df7a05b269a22b/code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm#L34
because it uses the new default value